sewing-kit.config.ts

```diff
@@ -11,11 +15,11 @@ export default function sewingKitConfig(
     name: 'polaris',
     library: true,
     plugins: [
-      plugins.jest((config) => {
+      plugins.jest((config: InitialOptions) => {
         config.roots = [join(__dirname, 'src'), join(__dirname, 'tests')];
 
         config.setupFiles.push(join(tests, 'setup.ts'));
-        config.setupTestFrameworkScriptFile = join(tests, 'each-test.ts');
```

Comment:
This is now deprecated and replaced with `setupFilesAfterEnv` which accepts multiple files